src/Entity.php

```diff
@@ -2100,500 +2090,77 @@ public static function showHelpdeskOptions(Entity $entity)
+                    return self::inheritedValue($item->getLink(), true, false);
                 }
-                self::inheritedValue($inherit, true);
-                echo "</td></tr>";
             }
+            return null;
+        };
 
-            echo "<tr class='tab_bg_1'><td colspan='4'>";
-
-            $_POST = [
-                ('inquest_config' . $config_suffix) => $entity->fields['inquest_config' . $config_suffix],
-                'entities_id' => $ID
-            ];
-            $params = [
-                ('inquest_config' . $config_suffix) => '__VALUE__',
-                'entities_id' => $ID
-            ];
-            echo "<div id='inquestconfig'>";
-            $ajax_file =  GLPI_ROOT . '/ajax/' . strtolower($inquest_itemtype::getType()) . '.php';
-            include $ajax_file;
-            echo "</div>\n";
-
-            echo "</td></tr>";
-        }
-
-        Plugin::doHook(Hooks::POST_ITEM_FORM, ['item' => $entity, 'options' => &$options]);
-
-        echo "</table>";
-
-        if ($canedit) {
-            echo "<div class='center'>";
-            echo "<input type='hidden' name='id' value='" . $entity->fields["id"] . "'>";
-            echo "<input type='submit' name='update' value=\"" . _sx('button', 'Save') . "\"
-                  class='btn btn-primary'>";
-            echo "</div>";
-            Html::closeForm();
-        }
-
-        echo "</div>";
-
-        Ajax::updateItemOnSelectEvent(
-            "dropdown_inquest_config$rand",
-            "inquestconfig",
-            $CFG_GLPI["root_doc"] . "/ajax/ticketsatisfaction.php",
-            $params
-        );
+        $crontask = new CronTask();
+        $inheritance_labels = [
+            'tickettemplates_id' => $fn_get_linked_inheritance_label(TicketTemplate::class),
+            'changetemplates_id' => $fn_get_linked_inheritance_label(ChangeTemplate::class),
+            'problemtemplates_id' => $fn_get_linked_inheritance_label(ProblemTemplate::class),
+            'calendars_id' => $fn_get_linked_inheritance_label(Calendar::class, __('24/7')),
+            'tickettype' => $fn_get_inheritance_label('tickettype'),
+            'auto_assign_mode' => $fn_get_inheritance_label('auto_assign_mode'),
+            'suppliers_as_private' => $fn_get_inheritance_label('suppliers_as_private'),
+            'anonymize_support_agents' => $fn_get_inheritance_label('anonymize_support_agents'),
+            'display_users_initials' => $fn_get_inheritance_label('display_users_initials'),
+            'contracts_id_default' => $fn_get_inheritance_label('contracts_id_default', 'contracts_strategy_default'),
+            'inquest_config' => $fn_get_inheritance_label('inquest_config'),
+            'inquest_config_change' => $fn_get_inheritance_label('inquest_config_change'),
+            'autoclose_delay' => $fn_get_inheritance_label('autoclose_delay'),
+            'autopurge_delay' => $fn_get_inheritance_label('autopurge_delay'),
```

Comment:
Call to `getUsedConfig()` was made with `default_value: self::CONFIG_NEVER` for this field. This is no longer the case, and I guess it may result in unexpected results.